### ISSUE TITLE
The text is not visible after a page refresh - report an add-on for abuse

### ISSUE THREAD

**Issue by ioanarusiczki** [mozilla/addons-frontend#6914]
Followup for https://github.com/mozilla/addons-frontend/issues/6883

STR:
1. Load AMO dev and login then open an add-on detail page
2. Click Report this add-on for abuse
3. Type in some text and then refresh page

Expected result:
After reopening the text box the text is still visible.

Actual result:
The text is no longer displayed.

Notes:
This issue is not reproducible with the collection notes, reviews or developer replies.
Reproducible on AMO dev - FF63(Win10). 

![not working with refresh](https://user-images.githubusercontent.com/33448286/48416863-f991fd00-e759-11e8-8696-91d44a35cbac.gif)

**Comment by willdurand** [comment]
Yep, this textarea is not auto-saved. This is a duplicate of #3279.

**Comment by ioanarusiczki** [comment]
@willdurand Alright. I filed it as a followup for https://github.com/mozilla/addons-frontend/issues/6883 because it's working now for the reviews and developer replies.
